index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="navbar">
+        <ul>
+             <li><a href="#about">About</a></li>
+             <li><a href="#projects">Work</a></li>
+             <li><a href="#contact">Contact</a></li>
+            
+        </ul>
+    </div>
```

Comment:
Please wrap the navbar inside a header for semantic purpose. it will make your code more readable and clean.